dockstore-webservice/src/main/java/io/dockstore/webservice/core/Event.java

```diff
@@ -31,7 +31,8 @@
 @Table(name = "event")
 @SuppressWarnings({"checkstyle:magicnumber", "checkstyle:hiddenfield"})
 @NamedQueries({
-        @NamedQuery(name = "io.dockstore.webservice.core.Event.findAllForOrganization", query = "SELECT eve FROM Event eve WHERE eve.organization.id = :organizationId ORDER BY dbCreateDate DESC")
+        @NamedQuery(name = "io.dockstore.webservice.core.Event.findAllForOrganization", query = "SELECT eve FROM Event eve WHERE eve.organization.id = :organizationId ORDER BY dbCreateDate DESC"),
```

Comment:
Wonder if sorting by database ID would be more efficient since createdate isn't indexed

Reply:
I think there could be some very rare edge cases where the sort order could be slightly different if I understand how it works -- createdate is set in Java code in two threads for two different events, but they are persisted to the database in different order.

But I think that would be rare enough and the times would effectively be the same anyway, so I agree that we should change it.

Should I go ahead and do so?

Reply:
Yes please, doesn't matter if its in this PR or the next